dispatcher.go

```diff
@@ -276,6 +294,7 @@ func (d *Dispatcher) Start() error {
 		allStarted []transport.Lifecycle
 	)
 
+	d.log.Info("Starting up.")
```

Comment:
Is this really the standard, capitalized with a period at the end? This is going to be ugly imo